src/components/App/App.jsx

```diff
@@ -1,14 +1,34 @@
+/* eslint-disable react/destructuring-assignment */
```

Comment:
**[OPTIONAL]** 
By disabling eslint rule, you risk missing out on the object of the rule.